src/roadmap/common.py

```diff
@@ -53,6 +60,21 @@ async def decode_header(
     return org_id
 
 
+async def decode_header_username(
+    x_rh_identity: t.Annotated[str | None, Header(include_in_schema=False)] = None,
+) -> str:
+    if x_rh_identity is None:
+        return ""
+
+    decoded_id_header = base64.b64decode(x_rh_identity).decode("utf-8")
+    id_header = json.loads(decoded_id_header)
+    identity = id_header.get("identity", {})
+    user = identity.get("user", "")
+    username = user.get("username", "")
+
+    return username
```

Comment:
I saw that there is other decode_header function that fetches the `org_id` happy to unify them into a single one that returns an object or dict with the org_id and username.